lua/lualine/themes/auto.lua

```diff
@@ -117,13 +117,14 @@ local colors = {
 -- Change brightness of colors
 -- darken incase of light theme lighten incase of dark theme
 
-if get_color_brightness(utils.extract_highlight_colors('Normal', 'bg')) > 0.5 then
+local normal_color = utils.extract_highlight_colors('Normal', 'bg')
+if normal_color ~= nil and get_color_brightness(normal_color) > 0.5 then
   brightness_modifier_parameter = -brightness_modifier_parameter
+  for name, color in pairs(colors) do
```

Comment:
No this is wrong . We need this to run when get_color_brightness return < 0.5 too .

Reply:
Sorry about that. I have now separated the if that checks if the normal color is not nil from the if that checks if the brightness modifier should be negated. The brightness modifier is now applied whenever normal_color ~= nil.